native/cocos/2d/renderer/Batcher2d.cpp

```diff
@@ -279,6 +290,7 @@ CC_FORCE_INLINE void Batcher2d::handleIADraw(RenderEntity* entity, RenderDrawInf
 
     UIMeshBuffer* currMeshBuffer = drawInfo->getMeshBuffer();
     currMeshBuffer->setDirty(true);
+    _currMeshBuffer = currMeshBuffer;
```

Comment:
```handleIADraw``` already has lot of  middle ware specific logic. So I think change name to `handleMiddleWareDraw` will be more clear and lead less misunderstanding